docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3211,7 +3225,9 @@ interactive session with the `exec` command.
     HTTP/1.1 200 OK
     Content-Type: application/vnd.docker.raw-stream
 
+    {% raw %}
```

Comment:
They are not lost but they are not always reliable when relying on just the nesting. Maybe @johndmulhausen remembers what this was about.